demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureToLegacyConverter.php

```diff
@@ -102,16 +103,23 @@ protected function convertOrgasToLegacy(Procedure $procedure): array
     {
         $nonPlanningOfficeOrganisationIds = $this->procedureRepository->getInvitedOrgaIds($procedure->getId());
         $planningOfficeIds = $this->procedureRepository->getPlanningOfficeIds($procedure->getId());
-        $isCustomerMasterBlueprint = $procedure->isCustomerMasterBlueprint();
+        $isCustomerMasterBlueprint =
+            $procedure->getId() === $procedure->getCustomer()?->getDefaultProcedureBlueprint()?->getId();
```

Comment:
See above, keep method

Reply:
restored in 6f731c5800aeab34e5de2da82e2fb5c1a96e133c